R/cookies.R

```diff
@@ -181,17 +177,10 @@ cookie_banner_server <- function(
       shiny::updateTabsetPanel(
         session = parent_session,
         "navlistPanel",
-        selected = "support_panel"
+        selected = "cookies_panel_ui"
```

Comment:
Just wondering if we should make this less of a direct dependency. Say if someone wanted to create a custom cookies panel, they could still use the banner and point it to their own panel in the navlist. We could maybe add an extra input to the function and give it `cookies_panel_ui` as a default, e.g.:
`   cookie_link_panel = "cookies_panel_ui"`
And then this line becomes:
`   selected = cookie_link_panel`